projects/storefrontapp-e2e-cypress/cypress/integration/regression/register.e2e-spec.ts

```diff
@@ -7,20 +8,23 @@ describe('Register', () => {
     cy.visit('/');
   });
 
-  // Behavior changed to automatic login.
-  it('should login when trying to register with the same email and correct password', () => {
+  it('should register and redirect to login page', () => {
```

Comment:
Same things as for mobile version of this test. Second test can be removed and this one could be simplified.